### DISCUSSION TITLE
I encountered an issue when using Vue 2 and trying to use a base64 encoded image. It throws an exception: 'TypeError: this.pendingRequests[e] is undefined.'

### DISCUSSION THREAD

**Discussion by sun-gq** [hrynko/vue-pdf-embed#171] [Q&A]
I encountered an issue when using Vue 2 and trying to use a base64 encoded image. It throws an exception: 'TypeError: this.pendingRequests[e] is undefined.' 
![图片](https://github.com/hrynko/vue-pdf-embed/assets/34677556/f20c60e1-e0fb-4a9e-a6e3-9100dcb75b18)

**Comment by hrynko** [comment]
Hi @sun-gq,

Could you create an example to reproduce?
